WooCommerce/Classes/ViewRelated/Survey/SurveyViewController.swift

```diff
@@ -69,33 +69,40 @@ extension SurveyViewController {
         case orderFormShippingLines
 
         fileprivate var url: URL {
-            switch self {
-            case .inAppFeedback:
-                return WooConstants.URLs.inAppFeedback
-                    .asURL()
-                    .tagPlatform("ios")
-                    .tagAppVersion(Bundle.main.bundleVersion())
-            case .productsFeedback:
-                return WooConstants.URLs.productsFeedback
-                    .asURL()
-                    .tagPlatform("ios")
-                    .tagAppVersion(Bundle.main.bundleVersion())
-            case .addOnsI1:
-                return WooConstants.URLs.orderAddOnI1Feedback
-                    .asURL()
-                    .tagPlatform("ios")
-                    .tagAppVersion(Bundle.main.bundleVersion())
-            case .orderCreation:
-                return WooConstants.URLs.orderCreationFeedback
-                    .asURL()
-                    .tagPlatform("ios")
-                    .tagAppVersion(Bundle.main.bundleVersion())
-            case .orderFormShippingLines:
-                return WooConstants.URLs.orderCreationShippingFeedback
-                    .asURL()
-                    .tagPlatform("ios")
-                    .tagAppVersion(Bundle.main.bundleVersion())
-            }
+            let url: URL = {
+                switch self {
+                case .inAppFeedback:
+                    return WooConstants.URLs.inAppFeedback
+                        .asURL()
+                        .tagPlatform("ios")
+                        .tagAppVersion(Bundle.main.bundleVersion())
+                case .productsFeedback:
+                    return WooConstants.URLs.productsFeedback
+                        .asURL()
+                        .tagPlatform("ios")
+                        .tagAppVersion(Bundle.main.bundleVersion())
+                case .addOnsI1:
+                    return WooConstants.URLs.orderAddOnI1Feedback
+                        .asURL()
+                        .tagPlatform("ios")
+                        .tagAppVersion(Bundle.main.bundleVersion())
+                case .orderCreation:
+                    return WooConstants.URLs.orderCreationFeedback
+                        .asURL()
+                        .tagPlatform("ios")
+                        .tagAppVersion(Bundle.main.bundleVersion())
+                case .orderFormShippingLines:
+                    return WooConstants.URLs.orderCreationShippingFeedback
+                        .asURL()
+                        .tagPlatform("ios")
+                        .tagAppVersion(Bundle.main.bundleVersion())
+                }
+            }()
+
+            let session = ServiceLocator.stores.sessionManager
+            return url.tagSiteInfo(siteID: session.defaultSite?.siteID,
+                                   storeUUID: session.defaultStoreUUID,
+                                   storeURL: session.defaultSite?.url)
```

Comment:
Non-blocking nit: Maybe we can set the `tagPlatform` and `tagAppVersion` here instead of repeating it for every URL?

Reply:
Good point. Done in https://github.com/woocommerce/woocommerce-ios/pull/15353/commits/0b949d24293ce0b1a0bdb7a305a56913eb3f0ca6